index.html

```diff
@@ -4,49 +4,250 @@
+            </div>            
+
+        </div>
+        <div class="action2">
+            <div class="normal-button4">
+                <button class="action-text4">see project</button>
+            </div>
         </div>
     </div>
+       
     </div>
-  </section>
+</div>
+<div class="about-myself">
+    <div class="top">
+        <h2 class="text-intro">About myself</h2>
+        <p class="text-description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of 
+            my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="social-media">
+                <div class="supporting-text">
+                    <h4>LET'S CONNECT</h4>
+                </div>
+                <ul class="network">
+                    <li><a href="https://twitter.com/AimeSebagabo"><i class='bx bxl-twitter'></i></a></li>
 
+                    <li><a href="https://www.linkedin.com/in/jean-aime-sebagabo-279aa2207/"></a><i class='bx bxl-linkedin'></i></a></li>
+
+                    <li><a href=""><i class='bx bxl-gmail'></i></a></li>
+
+                    <li><a href="https://github.com/Hazard10-eng"><i class='bx bxl-github'></i></a></li>
+                </ul>
+                <div class="resume">
+                   <input type="text" value="Get my resume" class="text-resume">
+                </div>
+                
+            </div>
+    </div>
+    <div class="skill-list">
+        <div class="frame12">
+            <div class="frame7">
+                <div class="separator-bottom"></div>
+                <p class="text-title">Languages</p>
+                <div class="normal-icon">
+                    <i class='bx bx-chevron-down'></i>
+                </div>
+                </div>
+                <div class="language-list">
+                    <div class="frame9">
+                        <img class="js-svg" src="./image/Javascript Ellipse.svg" alt="js">
+                        <p class="js-text">Javascript</p>
+                    </div>
+                    <div class="frame10">
+                        <img class="html-svg" src="./image/HTML Ellipse.svg" alt="HTML">
+                        <p class="html-text">HTML</p>
+                    </div>
+                    <div class="frame11">
+                        <img class="css-svg" src="./image/CSS Ellipse .svg" alt="css">
+                        <p class="css-text">CSS</p>
+                    </div>
+               
+            </div>
+        </div>
+        <div class="frame13">
+            <div class="separator-bottoms"></div>
+            <p class="text-titles">Frameworks</p>
+            <div class="normal-icons">
+                <i class='bx bx-chevron-right'></i>
+            </div>
+            </div>
+            <div class="frame14">
+                <p class="text-titlesk">Skills</p>
+                <div class="normal-iconsk">
```

Comment:
Please use `ul` and `li` for categories in the "About myself" section. `Languages`, `Frameworks`, and `Skills` should be list items.

```
<ul>
   <li>
       <p class="text-titles">Languages</p>
       . . . 
   </li>
   <li>
       <p class="text-titles">Frameworks</p>
       . . . 
   </li>
   <li>
       <p class="text-titles">Skills</p>
       . . . 
   </li>
</ul>
```